packages/client/src/client/App/Analytics/PnL/PNLBar.tsx

```diff
@@ -35,17 +33,33 @@ const getLogRatio: (max: number, numb: number) => number = (max, numb) => {
   return logNumb / logMax
 }
 
-const PNLBar = ({ symbol, basePnl, maxVal }: PNLBarProps) => {
+const PNLBar = ({
+  symbol,
+  profitOrLossValue,
+  largetProfitOrLossValue,
+}: PNLBarProps) => {
   const [hovering, setHovering] = useState(false)
-  const color = basePnl >= 0 ? "positive" : "negative"
+  const color = profitOrLossValue >= 0 ? "positive" : "negative"
```

Comment:
can we map colors in the theme, so we reference the pos/neg colors by name - I am doing this in RTEq MUI theme, as we did in UISK v1

Reply:
will clear your E2E failure too